hack/version-util.sh

```diff
@@ -6,13 +6,13 @@
 
 # git tag version extraction adapted from https://github.com/carvel-dev/imgpkg/blob/develop/hack/build-binaries.sh
 function get_latest_git_tag {
-  git describe --tags | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+(-alpha\.[0-9]+)?'
+  git tag -l | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+(-alpha\.[0-9]+)?' | sort -V | tail -1
 }
 
 function get_sgctrl_ver {
   echo "${1:-`get_latest_git_tag`}"
 }
 
 function get_sgctrl_ver_without_v {
-  git describe --tags | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+(-alpha\.[0-9]+)?'
+  git tag -l | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+(-alpha\.[0-9]+)?' | sort -V | tail -1
 }
```

Comment:
nit: It seems like there is an extra line in the end. Can we remove extra line ? 
Otherwise LGTM

Reply:
I think It's due to line missing I think golint expects to have new line at the end of the file

Reply:
But it was not earlier as well but I can add that